components/content/de/locale-chooser.md

```diff
@@ -0,0 +1,5 @@
+---
+help: 'Hilfen bei übersetzen'
```

Comment:
Although 'Hilf bei der Übersetzung' has an informal tone to it - "Bei der Übersetzung helfen" is a bit more formal #minor